bin/chandra_repro

```diff
@@ -2096,10 +2096,11 @@ def lookup_hrc_pi_filter(evtfile):
     calfiles = cdb.search
     
     if not calfiles or len(calfiles) == 0:
-        error_out("No TGPIMASK2 CALDB files found for this dataset")
+        v1("No TGPIMASK2 CALDB files found for this dataset")
```

Comment:
```suggestion
        v1("No TGPIMASK2 CALDB file found for this dataset")
```

Technically it's a single file, isn't it?

Reply:
updated.